src/py/flwr/client/app.py

```diff
@@ -388,6 +395,8 @@ def _load_client_app() -> ClientApp:
                 )
 
                 # Send
+                if not validate_out_message(out_message, in_metadata):
+                    log(ERROR, "Invalid out message: %s", out_message)
                 send(out_message)
```

Comment:
Should we still send the message if the validation fails? Or rather construct and send a different error message?